sad-process-db.py

```diff
@@ -14,9 +14,9 @@
 
 # Function to import the AICc results.
 def import_results(datafile):
-    """Imports raw result .csv files in the form: site, S, N, logseries, logseries_untruncated, pln, negbin, geometric, zipf."""
-    raw_results = np.genfromtxt(datafile, dtype = "S15, i8, i8, f8, f8, f8, f8, f8", skip_header = 1, 
-                                names = ['site', 'S', 'N', 'logseries', 'pln', 'negbin', 'geometric','AICc_zipf'], delimiter = ",", missing_values = '', filling_values = '')
+    """Imports raw result .csv files in the form: site, S, N, ogseries_untruncated, pln, negbin, zipf."""
```

Comment:
Looks like you deleted an extra `l` at the start of what is now `ogseries_untruncated` :smile: